src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -846,6 +846,12 @@ p≤q⇒0≤q-p {p} {q} p≤q = begin
   (↥ y ℤ.* ↥ v) ℤ.* (↧ x ℤ.* ↧ u) ∎)
   where open ≡-Reasoning; open ℤ-solver
 
+*-congʳ : ∀ p {q r} → q ≃ r  → p * q ≃ p * r
```

Comment:
Sorry just spotted that these proofs should have `RightCongruent` and `LeftCongruent` as types to ensure consistency with the rest of the library. Will need to be updated in the CHANGELOG as well.

Reply:
While making this change I realised that I had the names the wrong way around! (At least relative to `LeftCongruent` and `RightCongruent`. I can't remember what I referred to when I named them the first time round)

Reply:
Awesome, that's exactly why we have the standardise names :smile: